content/metadata.yaml

```diff
@@ -328,6 +338,18 @@ authors:
     affiliations:
       - Hackensack Meridian School of Medicine
       - Hackensack University Medical Center
+  - github: syzheng 
+    name: Siyuan Zheng
+    initials: SZ
+    email: zhengs3@uthscsa.edu
+    twitter: zhengsiyuan
+    orcid: 0000-0002-1031-9424
+    contributions:
+      - Formal analysis
+      - Writing - Original draft
+      - Supervision
+    affiliations:
+      - Greehey Children's Cancer Research Institute, UT Health San Antonio
```

Comment:
looks good to me! Thanks

Reply:
>   - github: NNoureen
    name: Nighat Noureen
    initials: NN
    orcid: 0000-0001-7495-8201
    email: noureen@uthscsa.edu
    contributions:
      - Formal analysis
      - Writing - Original draft
    affiliations: 
      - Greehey Children's Cancer Research Institute, UT Health San Antonio


It's okay.. Thanks